### DISCUSSION TITLE
Latest version doesn't post complete artifact

### DISCUSSION THREAD

**Discussion by mccrearyp** [zgosalvez/github-actions-report-lcov#197] [Q&A] [answered]
The latest version doesn't post the complete artifact. It posts the top level directory with the stats, but is missing the files needed to drill down into sub directories or files. If I run my code coverage locally I still get the files. Possibly related to this change?
![image](https://github.com/zgosalvez/github-actions-report-lcov/assets/899576/a3b74866-52b1-4b3b-ab92-f8842ce40ede)

**Comment by zgosalvez** [comment, marked as answer]
Thanks for bringing this up @mccrearyp! I created an issue for this and a fix in #200. Should now be available in the latest [v4.1.5](https://github.com/zgosalvez/github-actions-report-lcov/releases/tag/v4.1.5) release.

**Reply by mccrearyp** [reply]
Artifacts look good, thanks!
